src/Traveler.js

```diff
@@ -2,14 +2,11 @@ class Traveler {
   constructor(singleTraveler) {
     this.id = singleTraveler.id;
     this.name = singleTraveler.name;
-    this.travelerType = singleTraveler.travelerType;
```

Comment:
Just curious, why remove the travelerType attribute from constructor?

Reply:
Good question! I'm not using that data in this iteration, so I decided to take it out. If I end up using it down the line, I'll just add that piece back in.